src/remote.sh

```diff
@@ -113,23 +113,12 @@ function which_distro()
 # prepares such a directory.
 function prepare_host_deploy_dir()
 {
-  if [[ -z "$kw_dir" ]]; then
-    complain "\$kw_dir isn't set. The kw directory at home may not exist"
-    return 22
+  if [[ ! -d "$KW_CACHE_DIR/$LOCAL_REMOTE_DIR" ]]; then
+    mkdir "$KW_CACHE_DIR/$LOCAL_REMOTE_DIR"
   fi
 
-  # We should expect the setup.sh script create the directory $HOME/kw.
-  # However, does not hurt check for it and create in any case
```

Comment:
Did you remove the below lines due to the above comment?
I'm wondering if these lines can be useful for the case of global or package installation.

Reply:
Any comment?